### PR TITLE
feat: export diagnostics to csv

### PR DESCRIPTION
**Pull request by AndreyNegrov**
Related to task:
https://github.com/oat-sa/extension-tao-nec/pull/21

Description:
Export client diagnostics to csv file

How to test:
go to the client diagnostics page "/taoClientDiagnostic/Diagnostic/index" and click button 'export csv'

### CODE REVIEW

File: views/js/tools/csvExporter.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                    return result;
+                });
+            }
+
+            // a string variable is created containing the contents of the csv file
+            function arrayToCsv(data, columnDelimiter = ",", lineDelimiter = "\n") {
+                const keys = Object.keys(data[0]);
+                let result = "";
+                result += keys.join(columnDelimiter);
+                result += lineDelimiter;
```
Comment (ricproenca-tao) [inline views/js/tools/csvExporter.js]: I think you can do it in 1 line, right?
```let result = `${keys.join(columnDelimiter)}${lineDelimiter}`;```

File: views/js/tools/csvExporter.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                    result += lineDelimiter
+                });
+                return result
+            }
+
+            // file download from content
+            function downloadFile(content, filename, type) {
+                const blob = new Blob([content], {type: type});
+                const url = URL.createObjectURL(blob);
+                const link = $('<a></a>')
```
Comment (ricproenca-tao) [inline views/js/tools/csvExporter.js]: You are creating an object and you are not destroying it
I think we should delete this object after using it.
What do you think?

Something like this:
```
let link = $('<a></a>').attr('download', filename).attr('href', url);
link.get(0).click();
link = null;
```

**Comment by AndreyNegrov** [comment]
> Nice job, just a few things.

thanks

File: views/js/tools/csvExporter.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+            // file download from content
+            function downloadFile(content, filename, type) {
+                const blob = new Blob([content], {type: type});
+                const url = URL.createObjectURL(blob);
+                let link = $('<a></a>')
+                    .attr('download', filename)
+                    .attr('href', url)
+                    .get(0)
+                    .click();
```
Comment (ricproenca-tao) [inline views/js/tools/csvExporter.js]: `$('<a></a>').attr('download', filename).attr('href', url).get(0).click();` does not return the reference for the `<a>`. 
You need to create the reference first, then `click` and then delete...
Something like this:
```
let link = $('<a></a>');
link.attr('download', filename).attr('href', url).get(0).click();
....
link = null;
```

**Pull request merged**